### PR TITLE
fix: link displays entry-hyperlink id

### PR DESCRIPTION
**Pull request by dylanzheng94**
## ❓ Type of change
- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

## 📝 Checklist

- [x] I have linked an issue or discussion.

- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: app/pages/resource/_resource.vue [hunk 1]
```diff
@@ -152,61 +152,77 @@ export default {
...
+        renderNode: {
+          [INLINES.HYPERLINK]: (node) => {
+            return `<a class="text-blue-900 underline" href="${node.data.uri}">${node.content[0].value}</a>`
+          },
+          [INLINES.ENTRY_HYPERLINK]: (node) => {
+            const entry = entryLinks.get(node.data.target.sys.id)
+
+            if (entry.__typename === 'TestResource') {
+              const resourcePath = resourcePathPrefix + entry.urlSlug
+              return `<a class="text-blue-900 underline" href="${resourcePath}">${entry.title}</a>`
```
Comment (anikbrazeau) [inline app/pages/resource/_resource.vue:181]: I wonder whether we might leave the flexibility to have the text be a different string of text than the entry title? 

I'm thinking there might be instances where we want to write text and add a link but not necessarily carry over the title and/or its capitalization (for readability) and/or instances in French where the sentence might not make sense without changing a verb tense or gender/plural agreements.

**Comment by anikbrazeau** [comment]
Curious, will this activate functionality for internal Contentful "entries" as well as external "URLs" and will it work across content types/models: Resources, Topics, Subtopics, Collections?

File: app/pages/resource/_resource.vue (hunk 1 above)
Comment (dylanzheng94) [reply app/pages/resource/_resource.vue]: Hey really good catch! - I've made the change so that it will display the flexible string!

**Comment by dylanzheng94** [comment]
> Curious, will this activate functionality for internal Contentful "entries" as well as external "URLs" and will it work across content types/models: Resources, Topics, Subtopics, Collections?

What will work with the current fix is:
- external URLs
- Resources (Contentful entries)

I can enable the remains content types as well, while we're at it!

**Review by anikbrazeau — APPROVED** [review]
Tested the Amplify link and it looks good for the resource where we found the bug! :shipit:

**Pull request merged**